.github/workflows/release.yml

```diff
@@ -20,6 +20,8 @@ jobs:
         run: npm ci
       - name: Run tests
         run: npm test
+      - name: Regenerate docs
+        run: npm run docs
```

Comment:
Is this enough? Shouldn't the workflow commit the changes caused by the command? 🤔

Reply:
this step will handle it like a boss
https://github.com/asyncapi/parser-js/pull/62/files#diff-56c30d9224897b08c28075052d4595c5R43

Reply:
Perfect 😄